### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by apt105**
Another batch of small fixes to various pages. https://meshtastic-git-small-changes-apt105.vercel.app

### CODE REVIEW

File: website/docs/software/overview.md [hunk 1]
```diff
@@ -4,3 +4,22 @@ title: Overview
 sidebar_label: Overview
 slug: /software
 ---
+
+The following applications are available to support your Meshtastic network:
+  * The [firmware](/docs/software/device/device-software) to run on the devices
+  * Connect to the devices with our [Android app](/docs/software/android/android-installation)
+  * An [iOS application](/docs/software/ios/ios-development) is in the works
+  * [Meshtastic.js](/docs/software/js/getting-started) provides a javascript library to interface with devices
+  * [Meshtastic-python](/docs/software/python/python-installation) provides access from desktop computers
```
Comment (jfirwin) [inline website/docs/software/overview.md:13]: Should we mention the CLI is bundled in this package? Or add another bullet point for it and point it appropriately to something within software/meshtastic-python/cli (if that exists?).

File: website/docs/software/python/installation.md [hunk 2]
```diff
@@ -3,67 +3,121 @@ id: python-installation
...
-- Install meshtastic:
+* Install meshtastic:
     ```powershell
     pip3 install --upgrade meshtastic
-    ```
\ No newline at end of file
+    ```
+    
+</TabItem>
+</Tabs>
```
Comment (jfirwin) [inline website/docs/software/python/installation.md:123]: Love the use of tabs! And you kept the groupid the same, which will be great if they navigate to firmware where it'll be synced up!

**Review by jfirwin — COMMENTED** [review]
Everything looks great! My main comment was on the inclusion of a CLI bullet point for the software overview page. On mobile so if I missed anything that's why. The tabs look great!

File: website/docs/software/overview.md (hunk 1 above)
Comment (apt105) [reply website/docs/software/overview.md]: Good point, added.

Comment (jfirwin) [reply website/docs/software/overview.md]: Looks great!

**Comment by sachaw** [comment]
Could you please remove the about page entirely, it was left there originally to take some info from, but now serves little purpose.

**Comment by apt105** [comment]
About page removed

**Pull request merged**